rm-main/src/cli.rs

```diff
@@ -11,21 +11,48 @@ use crate::transmission;
 
 #[derive(Parser)]
 #[command(version, about)]
+#[derive(Debug)]
 pub struct Args {
     #[command(subcommand)]
     pub command: Option<Commands>,
 }
 
+#[derive(Debug)]
+pub enum ConfigType {
+    DefaultConfig,
+    DefaultKeymap
+}
+
 #[derive(Subcommand)]
+#[derive(Debug)]
 pub enum Commands {
     AddTorrent { torrent: String },
     FetchRss { url: String, filter: Option<String> },
+    PrintDefaultConfig { },
+    PrintDefaultKeyMap { }
 }
 
 pub async fn handle_command(config: &Config, command: Commands) -> Result<()> {
     match command {
         Commands::AddTorrent { torrent } => add_torrent(config, torrent).await?,
         Commands::FetchRss { url, filter } => fetch_rss(config, &url, filter.as_deref()).await?,
+        Commands::PrintDefaultConfig { } => print_config(ConfigType::DefaultConfig).await?,
+        Commands::PrintDefaultKeyMap { } => print_config(ConfigType::DefaultKeymap).await?,
```

Comment:
You can just do:
println!(rm_config::main_config::MainConfig::DEFAULT_CONFIG)
and for keymap:
println!(rm_config::keymap::KeymapConfig::DEFAULT_CONFIG)
but mark them as `pub` since they are private right now